frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs

```diff
@@ -150,6 +153,7 @@ uiAddVanityAccountSettings ideL mChainId initialNotes = Workflow $ do
             , _deploymentSettingsConfig_gasLimit = Nothing
             , _deploymentSettingsConfig_caps = Nothing
             , _deploymentSettingsConfig_extraSigners = []
+            , _deploymentSettingsConfig_includePreviewTab = False
```

Comment:
Sholud this also use `includePreviewTab` from above?

Reply:
It does now